components/dashboard/src/components/UsageBasedBillingConfig.tsx

```diff
@@ -199,9 +199,8 @@ export default function UsageBasedBillingConfig({ attributionId }: Props) {
         }
     };
 
-    const used = currentUsage >= 0 ? currentUsage : 0;
-    const totalAvailable = usageLimit + (currentUsage < 0 ? currentUsage * -1 : 0);
-    const percentage = Math.min(Math.max(Math.round((100 * used) / totalAvailable), 0), 100);
+    const balance = currentUsage * -1 + usageLimit;
```

Comment:
```suggestion
    const balance = usageLimit - currentUsage;
```

Reply:
I opted for the slightly more complex expression to get across that we are inverting the `currentUsage`.